packages/_site/src/pages/components/progressbar/index.tsx

```diff
@@ -83,7 +83,7 @@ export default function ProgressbarPage() {
                         <div>
                             <ExampleBlock
                                 title="Default"
-                                description=""
+                                description="Blue progress bar."
```

Comment:
A progress bar that is currently in progress and hasn't completed or encountered an error is blue.